postgres/src/test/scala/zio/sql/postgresql/FunctionDefSpec.scala

```diff
@@ -35,6 +37,37 @@ object FunctionDefSpec extends PostgresRunnableSpec with ShopSchema {
         r <- testResult.runCollect
       } yield assert(r.head)(equalTo(expected))
 
+      assertion.mapErrorCause(cause => Cause.stackless(cause.untraced))
+    },
+    testM("timeofday") {
+      val query = select(Timeofday()) from customers
+
+      val testResult = execute(query).to[String, String](identity)
+
+      val assertion =
+        for {
+          r <- testResult.runCollect
+        } yield assert(r.head)(
+          matchesRegex(
+            "[A-Za-z]{3}\\s[A-Za-z]{3}\\s[0-9]{2}\\s(2[0-3]|[01][0-9]):[0-5][0-9]:[0-5][0-9].[0-9]{6}\\s[0-9]{4}\\s[A-Za-z]{3}"
```

Comment:
I'll have to take your word for this :)

Reply:
that's my idea to test timeofday, because this function return everytime a precise instant, so it will be impossible to assert with equality... right? So I try with regex matching these cases